### PR TITLE
chatter:0.1.0

### PR DESCRIPTION
**Pull request by SylvanFranklin**
I am submitting
- [x] a new package
- [ ] an update for a package

*Chatter* is a dialog template designed for speed and ease of printing.  I use it mostly for simple translations, but it would also be great for screenplays or other theater related activities.  For me the most important feature is ergonomics and easy repeatability for a single class or assignment group. 

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

File: packages/preview/chatter/0.1.0/template/main.typ [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#import "../lib.typ" : *; #show: doc => conf(title: [My assignment \ name], doc)
```
Comment (elegaanz) [inline packages/preview/chatter/0.1.0/template/main.typ:1]:
```suggestion
#import "@preview/chatter:0.1.0" : *; #show: doc => conf(title: [My assignment \ name], doc)
```

File: packages/preview/chatter/0.1.0/lib.pdf
Comment (elegaanz) [inline packages/preview/chatter/0.1.0/lib.pdf:1]: Can you please delete this file?

File: packages/preview/chatter/0.1.0/template/main.typ [hunk 2]
```diff
@@ -0,0 +1,28 @@
+#import "@preview/chatter:0.1.0" : *; #show: doc => conf(title: [My assignment \ name], doc)
+
+#let a = "Sam"
+#let d = "Dog"
+
+#log(number_lines: 3)[
```
Comment (elegaanz) [inline packages/preview/chatter/0.1.0/template/main.typ:6]:
```suggestion
#log(number-lines: 3)[
```

File: packages/preview/chatter/0.1.0/README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Chatter
+Typst Dialogue template designed for print. `Chatter` Aims to be an ergonomic
+abstraction to the `typst` terms list function so that you can focus on
+writing. For the best experience use variables for your characters: 
+
+### Use 
+This will all be created for you with with `typst init`. 
+```typ
+#import "lib.typ" : *;
```
Comment (elegaanz) [inline packages/preview/chatter/0.1.0/README.md:9]:
```suggestion
#import "@preview/chatter:0.1.0" : *; 
```

File: packages/preview/chatter/0.1.0/lib.typ [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  }
+
+  #let character = (name) => [
+    #upper(name.slice(0, 3)).
+  ]
+
+  #let log(number-lines: 5, body) = {
+    set terms(separator: h(2em), hanging-indent: 5em, spacing: 1em)
+    set par.line(
+      numbering: i => if calc.rem(i, number_lines) == 0 {i},
```
Comment (elegaanz) [inline packages/preview/chatter/0.1.0/lib.typ:23]:
```suggestion
      numbering: i => if calc.rem(i, number-lines) == 0 {i},
```

**Review by elegaanz — COMMENTED** [review]
Sorry, there is another occurrence I missed.

**Comment by SylvanFranklin** [comment]
Sweet thanks!

**Comment by elegaanz** [comment]
Thank you!

**Pull request merged**